Part2Lesson1/Controllers/LessonController.cs

```diff
@@ -48,5 +83,52 @@ public IEnumerable<Lesson> GetLessons()
         {
             return _lessons;
         }
+
+        public IEnumerable<LessonRoom> GetLinkedPairsOfLessonAndRoom()
```

Comment:
The main problem of current method body is problem with opportunity to change LessonRoom objects outside controller.

Reply:
Good solution but please, don't mark comments of other people as resolved